client/index.html

```diff
@@ -1,9 +1,93 @@
-<html>
-    <head>
-    <script src="hi.js"></script>
-    <title>Hello world!</title>
-    </head>
-    <body>
-        <h1>A header</h1>
-    </body>
-</html>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<title>Login to Meatpocalypse</title>
+	<meta name="keywords" content="" />
+	<meta name="description" content="" />
+	<meta name="robots" content="index, follow" />
+	<meta charset="utf-8" />
+	<script type="text/javascript" src="./js/form.js"></script>
+	<link type="text/css" rel="stylesheet" href="css/style.css" media="screen" />
+	
+</head>
+<body>
+
+	<div id="wrapper">
+		<div id="wrappertop"></div>
+
+		<div id="wrappermiddle">
+
+			<h2 id="log">Login to Play!</h2>
+
+		<form id='myform' method='POST' action="/" target='formresponse'>
+			<div id="username_input">
+
+				<div id="username_inputleft"></div>
+
+				<div id="username_inputmiddle">
+					<input type="text" name="user" id="url" value="Username" onclick="this.value = ''">
+					<img id="url_user" src="./images/mailicon.png" alt="">
+				</div>
+
+				<div id="username_inputright"></div>
+
+			</div>
+
+			<div id="password_input">
+
+				<div id="password_inputleft"></div>
+
+				<div id="password_inputmiddle">
+					<input type="password" name="pass" id="url" value="Password" onclick="this.value = ''">
+					<img id="url_password" src="./images/passicon.png" alt="">
+				</div>
+
+				<div id="password_inputright"></div>
+
+			</div>
+
+			<div id="submit">
+				<input type="image" src="./images/submit_hover.png" id="submit1" value="Sign In">
+				<input type="image" src="./images/submit.png" id="submit2" value="Sign In">
+			</div>
+		</form>
+			
+		<h2 id="register" style="display: none;">Sign Up to Play!</h2>

```

Comment:
If you need this, can you put it somewhere else, this entire div is completely hidden?